gammapy/scripts/main.py

```diff
@@ -83,26 +85,75 @@ def cli_download(ctx, folder):
     ctx.obj = {"localfolder": folder}
 
 
+@cli.group('jupyter', short_help='Perform actions on notebooks')
+@click.option('--src', default='.',
+              help='Local folder or Jupyter notebook filename.')
+@click.pass_context
+def cli_jupyter(ctx, src):
+    """
+    Perform a series of actions on Jupyter notebooks.
+
+    The chosen action is applied for every Jupyter notebook present in the
+    current working directory. Option --file allows to chose a single file,
+    while option --fold allows to choose a different folder to scan. These
+    options are mutually exclusive, only one is allowed.
+
+    \b
+    Examples
+    --------
+    \b
+    $ gammapy jupyter stripout
+    $ gammapy jupyter --src=mynotebooks.ipynb execute
+    $ gammapy jupyter --src=myfolder/tutorials test
+    $ gammapy jupyter black
+    """
+    log = logging.getLogger(__name__)
+
+    paths = []
+    path = Path(src)
+    if not path.exists():
+        log.error("File or folder {} not found.".format(src))
+        sys.exit()
+
+    if path.is_dir():
+        for f in path.iterdir():
```

Comment:
I think this would be equivalent and a tiny bit shorter:
```
if path.is_dir():
    path = path.glob('*.ipynb')
```

Reply:
Probably making a list instead of a generator here is less error-prone (e.g. if it's looped over twice a generator will be empty on the second go):
```
if path.is_dir():
    paths = list(path.glob('*.ipynb'))
```